js/components/select.tsx

```diff
@@ -173,6 +179,20 @@ class Select extends React.Component<IItemsComponentProps, IDropDownState> {
     onOpenChange(isOpen) {
         if (!this.props.isLoading) {
             this.setState({ isOpen });
+            
+            const select = this.comboBoxRef.current;
+            const mainScroller = (select).closest('.main-scroller');
```

Comment:
Is this missing jQuery $? Don't think it will work in IE without it.

Reply:
Oh yeah, I'll reapply the JQuery marker and test in IE

Reply:
Done